mentor_classifier/mentor_classifier/api.py

```diff
@@ -194,24 +235,49 @@ def fetch_category(
 
 
 def generate_followups(
-    category: str, cookies: Dict[str, str] = {}, headers: Dict[str, str] = {}
+    category: str,
+    cookies: Dict[str, str] = {},
+    headers: Dict[str, str] = {},
 ) -> List[FollowupQuestion]:
     data = fetch_category(category, cookies=cookies, headers=headers)
     me = data.get("me")
     if me is None:
         raise NameError("me not found")
     category_answer = me.get("categoryAnswers", [])
-    recorded = [
+    category_answers = [
         AnswerInfo(
             answer_text=answer_data.get("answerText") or "",
             question_text=answer_data.get("questionText") or "",
         )
         for answer_data in category_answer
     ]
-    followups = NamedEntities(recorded).generate_questions()
+    all_answered = fetch_mentor_questions(cookies=cookies, headers=headers)
```

Comment:
under the covers, `fetch_mentor_questions` and `fetch_mentor_name` are making the same gql query twice. Let's consolidate this into a single gql query specific for what you need here and include *ONLY* the data you actually want in the GQL query (as opposed to a big generic ME query)

It would be nice to also roll that up into a single gql query w whatever `fetch_category`, but that is probably too much hassle for now